MC_1.10.2/build.gradle

```diff
@@ -0,0 +1,62 @@
+
+// For those who want the bleeding edge
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+        maven {
+            name = "forge"
+            url = "http://files.minecraftforge.net/maven"
+        }
+        maven {
+            name = "sonatype"
+            url = "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+        flatDir {
+            dirs 'libs'
+        }
+    }
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.2-SNAPSHOT'
+        classpath 'eu.appsatori:gradle-fatjar-plugin:0.2-rc1'
+        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.3'
+    }
+}
+apply plugin: 'com.github.johnrengelman.shadow'
+
+apply plugin: 'net.minecraftforge.gradle.forge'
+
+version = "1.10.2-1.0.0"
+group= "net.minecraftforge.mercurius"
+archivesBaseName = "Mercurius"
+
+minecraft {
+    version = "1.10.2-12.18.1.2075"
+    runDir = "run"
+    mappings = "snapshot_20160518"
```

Comment:
thats the same as the one currently on master. I dont wanna change it and fix any potential code breaks because of it.